src/build/sitemap.js

```diff
@@ -0,0 +1,25 @@
+const fs = require("fs").promises;
+const path = require("path");
+
+async function buildSitemap(pageIndex, buildOpts) {
```

Comment:
Nothing new except using the `pageIndex` instead of the old `metaIndex` to get a list of pages.